lib/aruba.py

```diff
@@ -94,6 +89,13 @@ def _TranslatePolicy(self, pol, exp_info):
       if 'ipv6' in filter_options:
         ip_ver = 6
       new_terms = []
+      for t in terms:
```

Comment:
I am guessing you are looping through twice to make it only log once. Have you given any thought to creating a logging filter? This could be useful in other areas of the code.